tests/suites/0_stateless/18_tpcds/18_0061_q61.sql

```diff
@@ -32,11 +31,9 @@ from
    and   ss_customer_sk= c_customer_sk
    and   ca_address_sk = c_current_addr_sk
    and   ss_item_sk = i_item_sk
-   and   ca_gmt_offset = -6
+   and   ca_gmt_offset = -0.06
```

Comment:
sqlogictest support template by {sf}.

Maybe we can add this in `databend-test.`